aws-datastore/src/main/java/com/amplifyframework/datastore/syncengine/PersistentMutationOutbox.java

```diff
@@ -226,6 +234,28 @@ public Completable markInFlight(@NonNull TimeBasedUuid pendingMutationId) {
         });
     }
 
+    /**
+     * Publish a successfully enqueued mutation to hub.
+     * @param pendingMutation A mutation that has been successfully enqueued to outbox
+     * @param <T> Type of model
+     */
+    private <T extends Model> void announceSuccessfulSave(PendingMutation<T> pendingMutation) {
+        Amplify.Hub.publish(
+            HubChannel.DATASTORE,
+            HubEvent.create(DataStoreChannelEventName.OUTBOX_MUTATION_ENQUEUED, pendingMutation)
```

Comment:
Similarly to OUTBOX_MUTATION_PROCESSED, just double check that the event payloads align.

```
{
    "model": "Note",
    "element": {
        "content": "content-1594938571139",
        "id": "d98f54ff-53dd-44d0-b99f-562e8f58ae3c"
    }
}
```